src/main/features/response/helpers/taskListBuilder.ts

```diff
@@ -80,7 +80,7 @@ export class TaskListBuilder {
 
   static buildSubmitSection (draft: ResponseDraft, externalId: string): TaskList {
     const tasks: TaskListItem[] = []
-    if (draft.isResponseRejectedFullyWithDispute() || draft.isResponseRejectedFullyWithAmountClaimedPaid()) {
+    if (draft.isDigitalResponseSelected()) {
```

Comment:
@varma02 your rightfully found this places as the case of the issue but solution is wrong. 

The reason it does not work as expected is that `draft.isResponseRejectedFullyWithDispute() || draft.isResponseRejectedFullyWithAmountClaimedPaid()` evaluates to false when response is not selected which is initial state. We would like to show `Check and send` task when response is not selected.